modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+----
+<1> The new machine, `clustername-8qw5l-master-3` is being created and is ready after the phase changes from `Provisioning` to `Running`.
++
+It should take a few minutes for the new machine to be created. The etcd cluster Operator will automatically sync when the machine or node returns to a healthy state.
+
+
+.. Verify that bmh becomes provisioned and no error reported:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+NAME                      STATE                  CONSUMER                       ONLINE ERROR AGE
+openshift-control-plane-0 externally provisioned examplecluster-control-plane-0 true         4h48m
+openshift-control-plane-1 externally provisioned examplecluster-control-plane-1 true         4h48m
+openshift-control-plane-2 provisioned            examplecluster-control-plane-3 true          47m
+openshift-compute-0       provisioned            examplecluster-compute-0       true         4h48m
+openshift-compute-1       provisioned            examplecluster-compute-1       true         4h48m
+----
+
+.. The new node is added and in ready state:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+$ oc get nodes
+NAME                     STATUS ROLES   AGE   VERSION
+openshift-control-plane-0 Ready master 4h26m v1.24.0+9546431
+openshift-control-plane-1 Ready master 4h26m v1.24.0+9546431
+openshift-control-plane-2 Ready master 12m   v1.24.0+9546431
+openshift-compute-0       Ready worker 3h58m v1.24.0+9546431
+openshift-compute-1       Ready worker 3h58m v1.24.0+9546431
+----
+
+.Verification
+
+. Verify that all etcd pods are running properly.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-etcd -o wide | grep etcd | grep -v guard
+----
++
+.Example output
+[source,terminal]
+----
+etcd-openshift-control-plane-0      5/5     Running     0     105m
+etcd-openshift-control-plane-1      5/5     Running     0     107m
+etcd-openshift-control-plane-2      5/5     Running     0     103m 
+----
++
+If the output from the previous command only lists two pods, you can manually force an etcd redeployment. In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
+----
++
+<1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
++
+.To verify there are exactly three etcd members, connect to the running etcd container, passing in the name of a pod that was not on the affected node:
++
+.In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
```

Comment:
```suggestion
. In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
```

Reply:
What...the...hale. I have corrected this. It was a small paragraph of direction that had gotten munged up with too many "+" signs....THANK YOU SO MUCH.